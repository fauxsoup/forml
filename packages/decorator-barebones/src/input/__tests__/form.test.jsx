import Form from '../form';
import React from 'react';
import { render } from '@testing-library/react';

describe('renders', function() {
    let localizer;
    test('its children', function() {
        const { container } = render(
            <Form value="test" />
        );

        expect(container).toMatchSnapshot();
        expect(container.querySelector('input')).not.toBeNull();
        expect(container.querySelector('input').value).toBe('test');
    })
})
