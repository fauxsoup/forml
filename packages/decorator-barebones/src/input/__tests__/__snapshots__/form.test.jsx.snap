// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders its children 1`] = `
<div>
  <input
    type="text"
    value="test"
  />
</div>
`;
